cbor.go

```diff
@@ -1,341 +1,41 @@
 package cose
 
-import (
-	"bytes"
-	"fmt"
-	"reflect"
-
-	"github.com/fxamacker/cbor/v2"
-	"github.com/pkg/errors"
-)
+import "github.com/fxamacker/cbor/v2"
 
+// CBOR Tags for COSE signatures registered in the IANA "CBOR Tags" registry.
+//
+// Reference: https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml#tags
 const (
-	// SignMessageCBORTag is the CBOR tag for a COSE SignMessage
-	// from https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml#tags
-	SignMessageCBORTag = 98
-
-	// Sign1MessageCBORTag is the CBOR tag for COSE Single Signer Data Object
-	Sign1MessageCBORTag = 18
+	CBORTagSignMessage  = 98
```

Comment:
Are these constants intended for user consumption? If not, we can hide them :)

Reply:
Yes, they are intended for user consumption.